### PR TITLE
Added Next/Previous Track MediaKeys Support

### PR DESCRIPTION
**Pull request by Barisukan**
Adds support to the Next/Previous Track MediaKeys which allows the user to go to the next/previous song from their keyboard with ease.

### CODE REVIEW

File: src/components/MusicPlayer/ThePlayerBar.vue [hunk 1]
```diff
@@ -393,6 +393,20 @@ It can be an album/playlist or the original song lists where this song is from
 
   created() {
     window.addEventListener('keydown', this.handleKeyDown);
+
+    //Add MediaKey Support using MediaSession
+    navigator.mediaSession.setActionHandler("nexttrack", _ => {
```
Comment (antrancs) [inline src/components/MusicPlayer/ThePlayerBar.vue]: Should check if mediaSession is available on navigator before using it

`if ('mediaSession' in navigator)`

More on this: https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API

Comment (antrancs) [inline src/components/MusicPlayer/ThePlayerBar.vue]: Got an ESLint error for "_" as its type is implicit. You can replace "_" with () => ....

File: src/components/MusicPlayer/ThePlayerBar.vue [hunk 2]
```diff
@@ -393,6 +393,20 @@ It can be an album/playlist or the original song lists where this song is from
...
+      if (this.canGoNext) {
+        this.playNext();
+      }
+    })
+
+    navigator.mediaSession.setActionHandler("previoustrack", _ => {
```
Comment (antrancs) [inline src/components/MusicPlayer/ThePlayerBar.vue]: Same as above: "_"

**Comment by antrancs** [comment]
Hi, thanks for the PR, there are few errors in the build because **mediaSession** is not supported in all browsers so I made a few comments to fix them.

One more thing is TypeScript does not support **mediaSession** yet, so you get errors when accessing it. A quick workaround is to add the declaration in the **shims-vue.d.ts** file

`interface Navigator {
  mediaSession: any;
}`

That way you get code completion when accessing **mediaSession**. 😃

**Comment by Barisukan** [comment]
Completely forgot that this runs on other browsers as well (was writing this using the chrome PWA, woops!) Went ahead and made the requested changes and it seemed to pass!

**Comment by antrancs** [comment]
Merged. Thanks 👍

**Pull request merged**